arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -5773,6 +5693,25 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		006A7868262F9F15001D2A5E /* XCRemoteSwiftPackageReference "arcgis-runtime-toolkit-ios" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/Esri/arcgis-runtime-toolkit-ios";
```

Comment:
Please note: Before we release 100.11.0 version of toolkit, this branch will show 
> ❌ Failed to resolve dependencies

This is expected because the release tag doesn't exist yet. We might need to test and merge this PR after toolkit is released.